### ISSUE TITLE
Support for ACLs/roles on service accounts

### ISSUE THREAD

**Issue by asadoughi** [pulumi/pulumi-confluent#10]
After creating a service account, I need to add roles/ACLs to the ServiceAccount, such as EnvironmentAdmin, such that it has access to the Kafka clusters that were created.

**Comment by leezen** [comment]
I believe [ACLs](https://www.pulumi.com/docs/reference/pkg/kafka/acl/) can be configured via the `kafka` provider per https://github.com/Mongey/terraform-provider-confluentcloud/issues/31

**Comment by aviflax** [comment]
Yup, that’s how I’m doing it. It’s a bit fiddly but I got it working.

**Comment by JimGalasyn** [comment]
@aviflax and @leezen, I need to provision ACLs, too -- is your solution to run Terraform separately to create ACLs and then run Pulumi? Or do you run ts2pulumi on a TF config file?
